_robots/2022Robot.md

```diff
@@ -15,22 +16,30 @@ features:
       icon: fa-crosshairs
 awardsWon: 4 
 hide_hero: true
+Competitions:
+- item: Heartland Regional
+- item: Central Missouri Regional
+- item: Hosted Greater Kansas City Regional
+- item: 2022 World Championships
+- item: Indiana Robotics Invitational
+- item: CowTown ThrowDown
 ---
 
-<h2>Robot capabilities</h2>
-* Four wheel highly maneuverable swerve drive capabilities.
-* Double high-speed climber arm capabilities. (Able to climb to the traversal rung, 7.58ft. in about 5.5 seconds)
-* High speed shooting capabilities of accurately scoring high ~95% of the time.
-* Triple roller bar collecting system
-* "S" shaped storage mechanism
-
-<h2>Robot facts</h2>
-* Weight: 119 lbs.
-* Two telescoping arms
-
-<h2>Competitions</h2>
-* Heartland Regional
-* Central Missouri Regional
-* Hosted Greater Kansas City Regional
-* 2022 World Championships
-* Indiana Robotics Invitational
+[//]: # (<h2>Robot capabilities</h2>)
+
+[//]: # (* Four wheel highly maneuverable swerve drive capabilities.)
+
+[//]: # (* Double high-speed climber arm capabilities. &#40;Able to climb to the traversal rung, 7.58ft. in about 5.5 seconds&#41;)
+
+[//]: # (* High speed shooting capabilities of accurately scoring high ~95% of the time.)
+
+[//]: # (* Triple roller bar collecting system)
+
+
+[//]: # ()
+[//]: # (<h2>Robot facts</h2>)
+
+[//]: # (* Weight: 119 lbs.)
+
+[//]: # (* Two telescoping arms)
```

Comment:
Why remove all of this content?

Reply:
Because it was apparently never re-formated :) Fixed now
https://github.com/FRCTeam1987/teambroncobots.com/pull/50/commits/f23594b3fcb32a5f7637a99288e87e3d1dd50c12